src/components/Stake/Stake.js

```diff
@@ -1,19 +1,57 @@
 import React from 'react'
 import { Header } from '@aragon/ui'
-import LayoutBox from '../Layout/LayoutBox'
+import { useStakingState } from '../../providers/Staking'
+import EmptyState from './EmptyState'
 import LayoutColumns from '../Layout/LayoutColumns'
 import LayoutGutter from '../Layout/LayoutGutter'
 import LayoutLimiter from '../Layout/LayoutLimiter'
 import SideBar from './SideBar'
+import StakingMovements from './StakingMovements'
+import noAccount from './assets/noAccount.svg'
 
 const Stake = React.memo(function Stake() {
+  const { stakeManagement } = useStakingState()
+
+  if (!stakeManagement) {
+    return (
+      <LayoutGutter>
+        <LayoutLimiter>
+          <Header primary="Stake management" />
+          <EmptyState
+            icon={noAccount}
+            title="Enable your account"
+            paragraph="Connect to an Ethereum provider to access your staking data. You may be temporarily redirected to a new screen."
```

Comment:
To what place? I think this is making referrence to the account module

Reply:
I figured it was referencing the provider explainer link (https://ethereum.org/en/wallets/) but it could also be an action to open the account module. Worth checking but probably not a big deal right now.